test-suite/src/snapshot_integration_tests.rs

```diff
@@ -22,42 +22,49 @@ pub fn writing_invalid_entity_returns_error() {
 pub fn create_and_read_snapshot() {
     let snapshot_path = env::temp_dir().join("test.snapshot");
 
-    let entity = get_test_entity().expect("Error");
+    let mut entity = get_test_entity();
 
     {
         SnapshotOutputStream::new(snapshot_path.clone())
-            .expect("Error")
-            .write_entity(EntityId::new(1), &entity)
-            .expect("Error");
+            .expect("Failed to create `SnapshotOutputStream`")
+            .write_entity(EntityId::new(1), &mut entity)
+            .expect("Failed to write entity to snapshot");
     }
 
     {
-        let mut snapshot = SnapshotInputStream::new(snapshot_path).expect("Error");
+        let mut snapshot = SnapshotInputStream::new(snapshot_path)
+            .expect("Failed to create `SnapshotInputStream`");
```

Comment:
Honestly embarrassed by these previous "error" messages 😬

Reply:
Hey, I'm just happy that you wrote the tests in the first place! I wouldn't have caught the error if we didn't have some tests in place 😁